card/lib/card/view/options.rb

```diff
@@ -136,9 +137,14 @@ def items
 
       # options to be used in data attributes of card slots (normalized options
       # with standard keys)
+      # FIXME: what we really want is options as they were when render was called.
+      # normalized is wrong because it can get changed before render. live is wrong
+      # because they can get changed after.  current solution is a compromise.
       # @return [Hash]
       def slot_options
-        normalized_options.select { |k, _v| Options.all_keys.include? k }
+        normalized_options.merge(view: requested_view).select do |k, _v|
+          Options.all_keys.include? k
+        end
```

Comment:
without this, a main view set in params (eg my_project?view=edit) is not listed as the view in a slot.  comment above explains further.